refract-spec.md

```diff
@@ -290,39 +293,37 @@ In JSON, an example of an object is:
 }
 ```
 
-In Refract, this object MAY Be expanded to include Member Elements as its content.
+In Refract, this object MAY Be expanded to include an array of elements as its content.
 
 ```json
 {
   "element": "object",
   "content": [
     {
-      "element": "member",
-      "content": {
-        "key": {
-          "element": "string",
-          "content": "foo"
-        },
-        "value": {
-          "element": "string",
-          "content": "bar"
-        }
-      }
+      "element": "string",
+      "meta": {
+        "name": "foo"
+      },
+      "content": "bar"
     }
   ]
 }
 ```
 
-### Member Element (Element)
+### Property Element (Element)
```

Comment:
This is mostly a constraint on any kind of element. This means that if an element definition calls for a Property Element, it MUST have a name defined for that element. If you don't do this, you have to rely on the description of name to tell you that it MUST be used when the element is used as a property.

So the only reason I like having this is that you surface this constraint to your MSON-aware clients and make the constraint machine-usable.